LandonHotel/Services/BookingService.cs

```diff
@@ -34,7 +34,22 @@ public bool IsBookingValid(int roomId, Booking booking)
         public double CalculateBookingCost(int roomId, Booking booking)
         {
             //TODO: Booking cost is the adjusted room rate times the number of days
-            return 0.0;
+            Room bookedRoom = _roomsRepo.GetRoom(roomId);
+            double guestNumRate;
+            //Checks to see if there are more than one guests
+            if(booking.NumberOfGuests == 1)
+            {
+                guestNumRate = 1;
```

Comment:
For consitency, this seems like it should be equal to the  **bookedRoom.Rate**
The room rate is  fixed by the number of guests